frontend/src/locales/da/input_fields.json

```diff
@@ -17,6 +17,8 @@
   "name": "Navn",
   "new_name": "Nyt navn",
   "org_unit_type": "@:shared.org_unit_type",
+  "user_key": "Enhedsnummer",
+  "user_key_placeholder": "Udfyld eller auto",
```

Comment:
"Udfyld eller auto" is not an especially helpful message

Reply:
Better than nothing - Alex loved it :-)

Reply:
Alright then. If it works for El Jefe, it works for me.